interface/drugs/add_edit_drug.php

```diff
@@ -617,18 +616,19 @@ function sel_related() {
 
 </table>
 <div class="btn-group">
-<input type='submit' class="btn btn-primary" name='form_save' value='<?php echo  $drug_id ? xla('Update') : xla('Add') ; ?>' />
-
-<?php if (AclMain::aclCheckCore('admin', 'super') && $drug_id) { ?>
-<input class="btn btn-danger" type='submit' name='form_delete' value='<?php echo xla('Delete'); ?>' />
-<?php } ?>
-<input type='button' class="btn btn-secondary" value='<?php echo xla('Cancel'); ?>' onclick='window.close()' />
+    <input type='submit' class="btn btn-primary" name='form_save' value='<?php echo  $drug_id ? xla('Update') : xla('Add') ; ?>' />
```

Comment:
lets not give up on the input to button conversion. Try this:
```
<button type='submit' class="btn btn-primary" name='form_save' value='<?php echo  $drug_id ? xla('Update') : xla('Add') ; ?>'><?php echo  $drug_id ? xla('Update') : xla('Add') ; ?></button>
```

Reply:
note this will then allow you to add a button class (such as btn-save) fontawesome icon also now that it is a button

Reply:
Thanks for your tips, I added value parameter it looks like working correctly.